slither/core/cfg/node.py

```diff
@@ -1046,6 +1049,43 @@ def __str__(self) -> str:
         txt = str(self._node_type.value) + additional_info
         return txt
 
+    def __getattr__(self, item: str):
```

Comment:
We should avoid this, and do a breaking change

Doing the __getattr__ trick will lead to more issues in the long term I think

Reply:
I've attempted to keep the compatibility layer minimal here to avoid future issues - however, I believe that implementing a complete breaking change will significantly simplify the code.